### ISSUE TITLE
[Feature] Add Daily Plan API

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#2391]
In Ever Gauzy, we need to create new DB tables and APIs for the below:

1. In Gauzy Database, we should have a record called "DailyPlan" that has ID, Date (i.e. for which date this plan), EmployeeId (who will work on this plan), WorkTimePlanned (how many hours/minutes employee plans to work), Status (TODO, IN PROGRESS, COMPLETED, note it's NOT same as issue status because here we just want to indicate if employee working on plan or not. I think for status we can just use some sort of Enum here that is SAME for all companies / tenants etc, i.e. hardcoded enum).

2. We also will have a records called "DailyPlanTasks" which will have DailyPlanTaskId autogenerated,  a "DailyPlanID" column (relation to DailyPlan ID column) and a "TaskId" column (relation to Tasks table ID column), so it essentially connects multiple tasks to a given daily plan.

3. APIs need to be built to CRUD daily plans for given employeeId (optionally filtered by Date)

Basically, we store for given employee for each day a daily plan record with some information and also we store what tasks are connected to that daily plan.  Of course, we only store for those days when employees created such plans, we don't store for every day of the year!
